.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+indent_size = 4
+
+[*.md]
+indent_size = 4
+trim_trailing_whitespace = false
+
+[*.yml]
```

Comment:
Is this section needed?  It looks like it duplicates what's in the `*` section

Reply:
I think this was just defensive in case the * changed.  Torn on whether that's redundant or good.